Scripts/Runtime/Jobs/JobTypes/AnvilJobBatchDeferExtension.cs

```diff
@@ -27,7 +25,8 @@ public static class AnvilJobBatchDeferExtension
             atomicSafetyHandlePtr = scheduleInfo.SafetyHandlePtr;
 #endif
 
-            IntPtr reflectionData = WrapperJobProducer<TJob>.GetReflectionData();
+            IntPtr reflectionData = WrapperJobProducer<TJob>.JOB_REFLECTION_DATA;
+            CheckReflectionDataCorrect(reflectionData);
```

Comment:
🚲 Suggest rename to `ValidateReflectionData()`

Reply:
Well that's what I get for copying Unity. Much better name, fixed.

Reply:
Is that seriously what they named theirs?! 🤦

Reply:
It sure is... it sadly sure is. Surprisingly one of the few things that's actually consistent between all job types too!